Cargo.toml

```diff
@@ -250,6 +258,7 @@ sources = [
   "sources-syslog",
   "sources-tls",
   "sources-vector",
+  "sources-kubernetes-logs",
```

Comment:
Please remove this until we have docs ready for the site. :)

Reply:
We might be fine with it being merged enabled without docs. Previous k8s impl was without docs for a while.
Addressing docs separately would really help reduce the overhead of changes at this PR, this is why I'm planning to address them as a separate PR. Plus it'd be easier to do when we have the whole picture (not just the source) implemented.